builder/llpcBuilderImpl.h

```diff
@@ -119,24 +119,218 @@ class BuilderImplMisc : virtual public BuilderImplBase
     BuilderImplMisc(llvm::LLVMContext& context) : BuilderImplBase(context) {}
 
     // Create a "kill". Only allowed in a fragment shader.
-    llvm::Instruction* CreateKill(const llvm::Twine& instName = "") override final;
+    llvm::Instruction* CreateKill(const llvm::Twine& instName) override final;
 
     // Create a "readclock".
-    llvm::Instruction* CreateReadClock(bool realtime, const llvm::Twine& instName = "") override final;
+    llvm::Instruction* CreateReadClock(bool realtime, const llvm::Twine& instName) override final;
 
 private:
     LLPC_DISALLOW_DEFAULT_CTOR(BuilderImplMisc)
     LLPC_DISALLOW_COPY_AND_ASSIGN(BuilderImplMisc)
 };
 
+class BuilderImplSubgroup : virtual public BuilderImplBase
+{
+public:
+    BuilderImplSubgroup(llvm::LLVMContext& context) : BuilderImplBase(context) {}
+
+    // Create a get subgroup size query.
+    llvm::Value* CreateGetSubgroupSize(const llvm::Twine& instName) override final;
+
```

Comment:
Where is the implementation of BuilderImplSubgroup? do you plan to submit it in next commit?

Reply:
https://github.com/GPUOpen-Drivers/llpc/pull/48/files#diff-dd074bfcc927dd88ea9427b56e709a31 - the file is over a certain size so you have to click the 'yes I really want to view this file' on the diff view!